libs/s25main/network/GameClient.cpp

```diff
@@ -1593,7 +1608,7 @@ void GameClient::SkipGF(unsigned gf, GameWorldView& gwv)
 
     unsigned start_ticks = VIDEODRIVER.GetTickCount();
 
-    if(!replayMode)
+    if(!replayMode && !aiBattleMode_)
```

Comment:
I think this won't work: Wouldn't the AI do nothing in this case (i.e. not in a replay)? The AI IIRC is handled on the server so simply running GFs will not let the server (and hence the AI) be involved.   
Can you verify this?

Reply:
Yes, you are right. Skipping won't work like this.